ramls/extended-error.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "An extended error",
+  "type": "object",
+  "properties": {
+    "message": {
+      "type": "string",
+      "description": "Error message text"
+    },
+    "type": {
+      "type": "string",
+      "description": "Error message type"
+    },
+    "code": {
+      "type": "string",
+      "description": "Error message code"
+    },
+    "parameters": {
+      "type": "object",
+      "description": "Error message parameters",
+      "$ref": "raml-util/schemas/parameters.schema"
+    },
+    "overridableBlock": {
+      "type": "object",
+      "description": "Details of an overridable block the error is related to",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the block",
+          "enum": [
+            "patronBlock",
+            "itemLimitBlock",
+            "itemNotLoanableBlock"
+          ]
+        },
```

Comment:
I think it is useful to clients that the API makes the relationship between an error and the mechanism to override it explicit